.github/workflows/release-please.yml

```diff
@@ -0,0 +1,13 @@
+ on:
+   push:
+     branches:
+      - main
+ name: release-please
+ jobs:
+   release-please:
+     runs-on: ubuntu-latest
+     steps:
+       - uses: google-github-actions/release-please-action@v2
+         with:
+           command: manifest
+           token: ${{ secrets.LOS_AUTO_BOT_RP_TOKEN }}
```

Comment:
copy/paste from sdk-codegen: enable the release-please action. note: there are a few different ways to configure release-please but I think that as a GH Action using the input `command: manifest` + a config and .manifest file is the most robust way

we should enable the "semantic pull requests" app/check on this repo to enforce conventional commit formatted commit messages (which release-please relies on for knowing how to bump versions)

Reply:
todo: for this and publish-metrics.yml we'll have to add this repo for access to the relevant org GitHub secrets
todo: I think we should only enable the "squash merge" setting (components and sdk-codegen have it, I think it should be our standard)